examples/CommandBasedRobot/src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -19,13 +25,67 @@
   // The controller for the swerve drive.
   private SwerveController swerve;
 
-  // The path to the swerve config file in the deploy directory.
-  private static final String SWERVE_CONFIG = "swerve.toml";
+  // Chooser for selecting the Joystick Control Style used
+  private final SendableChooser<String> oiChooser = new SendableChooser<>();
+
+  // Variables for control with different joystick setups
+  private double xDirection;
+  private double yDirection;
+  private double rotationSpeed;
+  private double translationSpeed;
 
   /** Constructor. */
   public Drivetrain() {
-    this.swerve = new SwerveController(SWERVE_CONFIG);
+    this.swerve = new SwerveController(DriveConstants.SWERVE_CONFIG);
     this.setYaw(0.);
+
+    // Creates a chooser dropdown with each of the driver control options
+    oiChooser.addOption("Split Joysticks with Triggers Turning", "Split Joysticks with Triggers Turning");
+    oiChooser.addOption("Single Joystick with Triggers Turning", "Single Joystick with Triggers Turning");
+    oiChooser.addOption("Single Joystick with Joystick X Turning", "Single Joystick with Joystick X Turning");
+    oiChooser.setDefaultOption("2 Joysticks with Gas Pedal", "2 Joysticks with Gas Pedal");
```

Comment:
Can we toss these strings in an enum so that there isn't as much copy-pasting?

Reply:
Do you mean copy-pasting in the code or on the user's side (the user shouldn't ever have to type out any of these)?  I think an enum would be just as verbose right?